### PR TITLE
Add bigtable GAPIC

### PR DESCRIPTION
**Pull request by andreamlin**
Adding generated directories:
- generated/java/gapic-google-cloud-bigtable-v2
- generated/java/gapic-google-cloud-bigtable-admin-v2
- generated/java/proto-google-cloud-bigtable-v2
- generated/java/proto-google-cloud-bigtable-admin-v2
- generated/java/grpc-google-cloud-bigtable-v2
- generated/java/grpc-google-cloud-bigtable-admin-v2

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL
I'm not sure why there's a golang test failure in DLP, given that no DLP files were changed/added/deleted.
The circle CI java test failure is due to a "WARNING" being thrown, but it isn't a test "ERROR," so that should be ok.

**Comment by andreamlin** [comment]
Reran CI and it's all green now!

**Comment by garrettjonesgoogle** [comment]
cc @igorbernstein2

File: generated/java/gapic-google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/v2/BigtableClient.java [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+ * </code>
+ * </pre>
+ */
+@Generated("by GAPIC v0.0.5")
+@BetaApi
+public class BigtableClient implements BackgroundResource {
+  private final BigtableSettings settings;
+  private final BigtableStub stub;
+
+  private static final PathTemplate TABLE_PATH_TEMPLATE =
```
Comment (neozwu) [inline generated/java/gapic-google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/v2/BigtableClient.java]: It seems to me you need to configure resource name class for these.

**Comment by igorbernstein2** [comment]
I would like to use static resource names. My only concern is that they would be duplicated & incompatible with cloud-bigtable-admin

**Comment by neozwu** [comment]
@igorbernstein2 hmmm, my guess is If two proto resource classes with same name from bigtable and bigtable/admin/ were imported to the same java file, toolkit should be able to resolve it correctly.

**Comment by igorbernstein2** [comment]
When I generated the admin, the TableName ended up as `com.google.bigtable.admin.v2.TableName`, I would imagine that enabling it here would generate it as `com.google.bigtable.v2.TableName`. Which I guess can be worked around by renaming the admin's package_name to `com.google.bigtable.v2`. But the we would run into the problem that you mentioned about the conflicting `package-info.java` file.

I'm starting to think that we should just keep the admin & data clients separate.

**Comment by igorbernstein2** [comment]
In either case I created a CL to generate the type safe names

File: generated/java/gapic-google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/v2/BigtableClient.java (hunk 1 above)
Comment (andreamlin) [reply generated/java/gapic-google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/v2/BigtableClient.java]: Blocked on cl/173174433.

Comment (andreamlin) [reply generated/java/gapic-google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/v2/BigtableClient.java]: Fixed.

**Comment by andreamlin** [comment]
Updated the generated code with ResourceNames, reflecting the updated bigtable_gapic.yaml.
PTAL

**Pull request merged**